src/references/index.ts

```diff
@@ -3,7 +3,7 @@ import { savingsAssets } from './compound';
 import { default as DefaultTokenListsSource } from './default-token-lists.json';
 import { Asset, SavingsAsset } from '@rainbow-me/entities';
 import { Network } from '@rainbow-me/helpers/networkTypes';
-
+export { default as polygonAllowList } from './polygon-allowlist.json';
```

Comment:
```suggestion
export { default as polygonAllowList } from './polygon-allowlist';
```

isn't this enough? (not sure)